ktor-server/ktor-server-netty/jvm/test/io/ktor/tests/server/netty/EngineMainTest.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.tests.server.netty
+
+import com.typesafe.config.ConfigFactory
+import io.ktor.server.config.*
+import io.ktor.server.netty.*
+import io.ktor.server.netty.EngineMain.loadConfiguration
+import io.netty.handler.codec.http.*
+import org.junit.*
+import org.junit.Assert.*
+
+class EngineMainTest {
+
+    @Test
+    fun testNettyCodecConfiguration() {
+        val config = HoconApplicationConfig(ConfigFactory.parseString("""
```

Comment:
Codestyle check fails on this line. Can you put parameter on new lines here and in the similar lines?
or run `./gradlew formatKotlin` to fix it automatically.